[update] updating email docs regarding customizations

angelaLazar opened the pull request:
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [x] Commit messages conforms to our [commit message standards](https://github.com/auth0/docs#commit-messages)
- [x] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [x] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 
### Description

Please describe your pull request.

Thank you!

angelaLazar committed:
[update] updating email docs regarding customizations

angelaLazar committed:
[update] updating email docs regarding customizations

angelaLazar committed:
[update] updating email docs regarding customizations

angelaLazar committed:
[update] adding migration info for email changes

angelaLazar committed:
[update] adding migration info for email changes

rolodato committed:
[bug] Drop the "the", it's cleaner

Annyv2 committed:
[edit] Typo

woloski committed:
move fb & twitter to optional section

woloski committed:
native integrations as optional

charlierudolph committed:
[bug] fix internal event typo

ntotten commented on articles/email/index.md:
```diff
@@ -5,172 +5,27 @@ url: /email
-* `{user.email}`
-
-If the **Subject** field is empty, Auth0 will auto-populate this text depending on what type of email you're sending. For example, one subject line might be "*Verify your email.*"
+Auth0's built-in email provider will no longer be supported for use in a production environment. 
```
lets change this to say "is not supported" since it is already in effect.

ntotten commented on articles/email/index.md:
```diff
@@ -5,172 +5,27 @@ url: /email
+:::
 
-If a user attempts and fails to log in ten or more times from a specific IP address, the user's account becomes locked and they will receive a **Blocked Account** email. Once the user receives this email, they will not be able to log in from that IP address again until they click on the link contained in the email.
+Auth0 provides built-in email services to easily communicate with your users. This includes, verification emails, welcome emails, change password emails and blocked account emails.
```
Extra comma `This includes, verification`

ntotten replied:
Also I prefer the trailing comma: `change password emails, and blocked account emails.`

ntotten commented on articles/email/index.md:
```diff
@@ -5,172 +5,27 @@ url: /email
+Auth0 provides built-in email services to easily communicate with your users. This includes, verification emails, welcome emails, change password emails and blocked account emails.
 
-If the user successfully logs in before they exhaust their ten allowed attempts, the counter is reset.
+When you first create your application, Auth0 provides a built-in email provider to send emails. This is meant to be used for testing purposes only. When using this, it is important to note:
```
>  application, Auth0 provides a 

no comma

angelaLazar committed:
[update] updating email docs

angelaLazar committed:
[update] updating email docs

angelaLazar commented:
addressed comments

ntotten commented on updates/2016-04-05.yml:
```diff
@@ -0,0 +1,14 @@
+      Updated the page to reflect new rules on using Auth0's built in email provider. Moved content on customizing emails to a new page.
+added:
+  -
+    title: "Customizing Your Emails"
```
We dont need to be this detailed with the updates. Just make a single update that describes all the changes to the the email content.

angelaLazar committed:
shortening update

angelaLazar committed:
shortening update

eugeniop committed:
Edits across the board

* Edits to Kerberos. Added article on Progressive profiling

* Fixed diagram on Progressive Profiling article

* Multiple edits on various articles

* Minor edits/typos

angelaLazar committed:
[edit] update with new SSO button name

ntotten committed:
Deleted configs

gnandretta committed:
[bug] Fix links in Lock 10 page to secondary pages

gnandretta committed:
[bug] Lock 10 page fixes

- Corrects information about release plan.
- Corrects information about foreign languages translations.
- Corrects information about customization.
- Corrects options names.

gnandretta committed:
[edit] Change wording about what's new in Lock 10

gnandretta committed:
[bug] Lock v10 installation page fixes

- Corrects version specification instruction.
- Corrects CDN script url.
- Corrects bower installation command.
- Corrects npm installation command.
- Corrects links to bundling examples.
- Corrects comments in samples for implementing redirect and popup mode.
- Corrects information in show sample.

Commit:
[update] Add info on multi-language funcationality for Passwordless SMS

gnandretta committed:
[new] Migration guide to Lock v10 in install page

- Provide instructions for new users an users that are upgrading.
- Detail what has changed.

gnandretta committed:
[bug] Fix Lock v10 new features page samples

Mostly update option names.

gnandretta committed:
[new] Complete Lock v10 new features page

- Add custom sign up fields sample.
- Add custom avatar provider sample.
- Add prefilled fields sample.

ntotten committed:
Merge pull request #1145 from gnandretta/lock-v10-refinements

Lock v10 refinements

Commit:
[update] Add info on custom fields

ntotten committed:
Merge pull request #1144 from khorne3/passwordless-sms

[update] Add info on multi-language funcationality for Passwordless SMS

ntotten committed:
Merge pull request #1146 from khorne3/custom-fields

[update] Add info on custom fields

angelaLazar committed:
fixing merge conflicts
